zerver/tests/fixtures/markdown_test_cases.json

```diff
@@ -421,10 +421,10 @@
     },
     {
       "name": "two_inline_images",
-      "input": "Google logo today: https://www.google.com/images/srpr/logo4w.png\nKinda boring\nZulip logo: https://zulip.com/static/images/landing-page/zulip-octopus.png",
-      "expected_output": "<p>Google logo today: <a href=\"https:\/\/www.google.com\/images\/srpr\/logo4w.png\">https:\/\/www.google.com\/images\/srpr\/logo4w.png<\/a><br>\nKinda boring<br>\nZulip logo: <a href=\"https:\/\/zulip.com\/static\/images\/landing-page\/zulip-octopus.png\">https:\/\/zulip.com\/static\/images\/landing-page\/zulip-octopus.png<\/a><\/p>\n<div class=\"message_inline_image\"><a href=\"https:\/\/www.google.com\/images\/srpr\/logo4w.png\"><img data-src-fullsize=\"\/thumbnail?url=https%3A%2F%2Fwww.google.com%2Fimages%2Fsrpr%2Flogo4w.png&amp;size=full\" src=\"\/thumbnail?url=https%3A%2F%2Fwww.google.com%2Fimages%2Fsrpr%2Flogo4w.png&amp;size=thumbnail\"><\/a><\/div><div class=\"message_inline_image\"><a href=\"https:\/\/zulip.com\/static\/images\/landing-page\/zulip-octopus.png\"><img data-src-fullsize=\"\/thumbnail?url=https%3A%2F%2Fzulip.com%2Fstatic%2Fimages%2Flanding-page%2Fzulip-octopus.png&amp;size=full\" src=\"\/thumbnail?url=https%3A%2F%2Fzulip.com%2Fstatic%2Fimages%2Flanding-page%2Fzulip-octopus.png&amp;size=thumbnail\"><\/a><\/div>",
+      "input": "Google logo today: https://www.google.com/images/srpr/logo4w.png\nKinda boring\nZulip logo: https://zulip.com/static/images/characters/zulip-octopus.png",
+      "expected_output": "<p>Google logo today: <a href=\"https:\/\/www.google.com\/images\/srpr\/logo4w.png\">https:\/\/www.google.com\/images\/srpr\/logo4w.png<\/a><br>\nKinda boring<br>\nZulip logo: <a href=\"https:\/\/zulip.com\/static\/images\/characters\/zulip-octopus.png\">https:\/\/zulip.com\/static\/images\/characters\/zulip-octopus.png<\/a><\/p>\n<div class=\"message_inline_image\"><a href=\"https:\/\/www.google.com\/images\/srpr\/logo4w.png\"><img data-src-fullsize=\"\/thumbnail?url=https%3A%2F%2Fwww.google.com%2Fimages%2Fsrpr%2Flogo4w.png&amp;size=full\" src=\"\/thumbnail?url=https%3A%2F%2Fwww.google.com%2Fimages%2Fsrpr%2Flogo4w.png&amp;size=thumbnail\"><\/a><\/div><div class=\"message_inline_image\"><a href=\"https:\/\/zulip.com\/static\/images\/characters\/zulip-octopus.png\"><img data-src-fullsize=\"\/thumbnail?url=https%3A%2F%2Fzulip.com%2Fstatic%2Fimages%2Fcharacters%2Fzulip-octopus.png&amp;size=full\" src=\"\/thumbnail?url=https%3A%2F%2Fzulip.com%2Fstatic%2Fimages%2Fcharacters%2Fzulip-octopus.png&amp;size=thumbnail\"><\/a><\/div>",
       "backend_only_rendering": true,
-      "text_content": "Google logo today: https:\/\/www.google.com\/images\/srpr\/logo4w.png\nKinda boring\nZulip logo: https:\/\/zulip.com\/static\/images\/landing-page\/zulip-octopus.png\n"
+      "text_content": "Google logo today: https:\/\/www.google.com\/images\/srpr\/logo4w.png\nKinda boring\nZulip logo: https:\/\/zulip.com\/static\/images\/characters\/zulip-octopus.png\n"
```

Comment:
I think in this case it'd be better to just change the test to any other random image from `static/images/` that we expect to be stable over time, like the Zulip logo, which the text in this test even inaccurately suggests is what this is for some historical reason.

Reply:
Makes sense. I went with `zulip-icon-128x128.png` in the `/static/images/logo` directory as it's also used in a puppeteer test, `admin.ts`. And `zulip-octopus.png` stays in the landing-page image directory.

Reply:
Sounds good!